cpp/src/io/parquet/page_data.cu

```diff
@@ -872,15 +872,14 @@ static __device__ bool setupLocalPageInfo(page_state_s* const s,
         case BOOLEAN:
           s->dtype_len = 1;  // Boolean are stored as 1 byte on the output
           break;
-        case INT32:
+        case INT32: [[fallthrough]];
         case FLOAT: s->dtype_len = 4; break;
         case INT64:
           if (s->col.ts_clock_rate) {
             int32_t units = 0;
-            if (s->col.converted_type == TIME_MILLIS or s->col.converted_type == TIMESTAMP_MILLIS) {
+            if (s->col.converted_type == TIMESTAMP_MILLIS) {
               units = cudf::timestamp_ms::period::den;
-            } else if (s->col.converted_type == TIME_MICROS or
-                       s->col.converted_type == TIMESTAMP_MICROS) {
+            } else if (s->col.converted_type == TIMESTAMP_MICROS) {
```

Comment:
DURATION types excluded because we never scale when reading - timestamp_type does not apply any more.